CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ Summary
 
 Fixed
 ^^^^^
+* Removed the ``ValueError`` raised by :attr:`~.HotRegion.__init__` ``if not phase_bounds or None in phase_bounds`` so that user can set a derived ``phase_shift`` value.
```

Comment:
if you format this file with Sphinx by rebuilding the doc pages, I believe you will find some problems with the cross-referencing to the API.

also general tidying like adding add necessary punctuation would be appreciated for readability.